src/utils/CustomActions.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Custom Actions
+ * ===================
+ *
+ * Stores code for custom actions to be taken during conversations.
+ */
+
+'use strict';
+
+module.exports = CustomActions;
+
+function CustomActions(game/*, ...args*/) {
+  Phaser.Group.call(this, game/*, ...args*/);
+  this._game = game;
+}
+
+CustomActions.prototype = Object.create(Phaser.Group.prototype);
+CustomActions.prototype.constructor = CustomActions;
+
+CustomActions.prototype.customAction = function(action) {
+  if (action.startsWith('fadeInAndOut')) {
+  	this.fadeInAndOut(parseInt(action.substring('fadeInAndOut'.length)));
+  }
+  else if (action === 'startConv:prologue02') {
+    this._game.camera.fade('#000000', 2000);
+    this._game.camera.onFadeComplete.addOnce(function() {
+      this._game.camera.flash('#000000', 2000);
+      this.startConversation('prologue02');
+    }, this);
+  }
+};
```

Comment:
Got it. We can talk about it on Saturday as well–I imagine there should be a set of functions which are fairly generalizable. I also just needed to get these changes merged in for my own progress.